telegram/chat.py

```diff
@@ -86,32 +91,32 @@ class Chat(TelegramObject):
 
     """
 
-    PRIVATE = 'private'
+    PRIVATE: str = 'private'
     """:obj:`str`: 'private'"""
-    GROUP = 'group'
+    GROUP: str = 'group'
     """:obj:`str`: 'group'"""
-    SUPERGROUP = 'supergroup'
+    SUPERGROUP: str = 'supergroup'
     """:obj:`str`: 'supergroup'"""
-    CHANNEL = 'channel'
+    CHANNEL: str = 'channel'
     """:obj:`str`: 'channel'"""
 
     def __init__(self,
-                 id,
-                 type,
-                 title=None,
-                 username=None,
-                 first_name=None,
-                 last_name=None,
-                 bot=None,
-                 photo=None,
-                 description=None,
-                 invite_link=None,
-                 pinned_message=None,
-                 permissions=None,
-                 sticker_set_name=None,
-                 can_set_sticker_set=None,
-                 slow_mode_delay=None,
-                 **kwargs):
+                 id: int,
+                 type: str,
+                 title: str = None,
+                 username: str = None,
+                 first_name: str = None,
+                 last_name: str = None,
+                 bot: 'Bot' = None,
+                 photo: ChatPhoto = None,
+                 description: str = None,
+                 invite_link: str = None,
+                 pinned_message: 'Message' = None,
+                 permissions: ChatPermissions = None,
+                 sticker_set_name: str = None,
+                 can_set_sticker_set: bool = None,
+                 slow_mode_delay: int = None,
+                 **kwargs: Any):
```

Comment:
I'd generally prefer `**kwargs: Dict[str, Any]`

Reply:
https://www.python.org/dev/peps/pep-0484/#arbitrary-argument-lists-and-default-argument-values

Reply:
Uiiii TIL, thanks!